### PR TITLE
New icon: Filter

### PR DESCRIPTION
**Pull request by jameskoster**
Discussed here https://github.com/Automattic/gridicons/pull/177

### CODE REVIEW

**Comment by drw158** [comment]
seems like this is good to go

File: docs/index.html [hunk 1]
```diff
@@ -76,7 +76,6 @@
         rows[i].onclick = function () {
             var cssClass = this.getAttribute( 'class' );
             var iconID = cssClass.split(' gridicons-')[1];
-			alert(iconID);
```
Comment (drw158) [inline docs/index.html]: What is this and why is it removed?

Comment (jameskoster) [inline docs/index.html]: Good question, honestly I don't know. That file (and svg-sprite/index.html) are generated by a grunt task I believe.

I'll look into it later but probably safest to just remove.

Comment (drw158) [inline docs/index.html]: I know there were recent changes to the gridicons page, it was probably related to that. Probably fine to let the grunt task to its thing.

**Comment by drw158** [comment]
Sorry @jameskoster I'm going to go ahead and include this icon in a bigger PR. It'll be easier than trying to coordinate our PRs. #187

**Pull request closed without merging**